### ISSUE TITLE
[Bug]: zsh completion for manual pages is not working

### ISSUE THREAD

**Issue by ELWAER-M** [termux/termux-packages#12073]
### Problem description

zsh completion for manual pages is not working, as far as i know this feature is part of `compinit` system that zsh have, i think zsh have hardcoded path for manual pages maybe?

### What steps will reproduce the bug?

```
man nv<Tab>
# yes i have nvim installed, and its manual page is exist
```

### What is the expected behavior?

complete the manual page name

### System information

termux-info:

```
Termux Variables:
TERMUX_API_APP__VERSION_NAME=0.50.1
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP__AM_SOCKET_SERVER_ENABLED=true
TERMUX_APP__APK_PATH=/data/app/~~W458SgnYh4hjTEWtspVRTw==/com.termux-GajMVwF6JHAytDPIym7T6A==/base.apk
TERMUX_APP__APK_RELEASE=GITHUB
TERMUX_APP__FILES_DIR=/data/user/0/com.termux/files
TERMUX_APP__IS_DEBUGGABLE_BUILD=true
TERMUX_APP__IS_INSTALLED_ON_EXTERNAL_STORAGE=false
TERMUX_APP__PACKAGE_MANAGER=apt
TERMUX_APP__PACKAGE_NAME=com.termux
TERMUX_APP__PACKAGE_VARIANT=apt-android-7
TERMUX_APP__PID=25329
TERMUX_APP__SE_FILE_CONTEXT=u:object_r:app_data_file:s0:c22,c257,c512,c768
TERMUX_APP__SE_INFO=default:targetSdkVersion=28:complete
TERMUX_APP__SE_PROCESS_CONTEXT=u:r:untrusted_app_27:s0:c22,c257,c512,c768
TERMUX_APP__TARGET_SDK=28
TERMUX_APP__UID=10278
TERMUX_APP__USER_ID=0
TERMUX_APP__VERSION_CODE=118
TERMUX_APP__VERSION_NAME=0.118.0+1ce120e
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0+1ce120e
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages-cf.termux.dev/apt/termux-main stable main
# x11-repo (sources.list.d/x11.list)
deb https://packages-cf.termux.dev/apt/termux-x11 x11 main
# tur-repo (sources.list.d/tur.list)
deb https://tur.kcubeterm.com tur-packages tur tur-on-device
# root-repo (sources.list.d/root.list)
deb https://packages-cf.termux.dev/apt/termux-root root stable
Updatable packages:
All packages up to date
termux-tools version:
1.29.2
Android version:
11
Kernel build information:
Linux localhost 4.4.177-21848546 #1 SMP PREEMPT Tue Jun 8 20:43:28 +07 2021 aarch64 Android
Device manufacturer:
samsung
Device model:
SM-A205F
```

**Comment by errr-maxx** [comment]
Workaround is `export MANPATH=$PREFIX/share/man`

**Comment by licy183** [comment]
Fixed by #12089
